### PR TITLE
Add data filter example to the website

### PR DESCRIPTION
**Pull request by Pessimistress**
data-filter-example

#### Change List
- Add data filter example

### CODE REVIEW

File: examples/website/data-filter/app.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+          />
+        )}
+      </Fragment>
+    );
+  }
+}
+
+export function renderToDOM(container) {
+  render(<App />, container);
+  require('d3-request').csv(DATA_URL, (error, response) => {
```
Comment (xintongxia) [inline examples/website/data-filter/app.js]: Cannot use `loaders.gl`?

File: examples/website/data-filter/range-input.js [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+      newValueMin = min;
+    }
+    this.props.onChange({
+      value: [newValueMin, newValueMin + span]
+    });
+
+    this._animationFrame = requestAnimationFrame(this._animate);
+  }
+
+  _renderThumbValue({$thumbIndex, $value}) {
```
Comment (xintongxia) [inline examples/website/data-filter/range-input.js]: what does `$` represent?

Comment (Pessimistress) [reply examples/website/data-filter/range-input.js]: That's the naming convention that baseui uses.

File: examples/website/data-filter/app.js (hunk 1 above)
Comment (Pessimistress) [reply examples/website/data-filter/app.js]: Loaders' CSV module only supports async iterators.

File: website/src/components/demos/data-filter-demo.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import React, {Component} from 'react';
+import {readableInteger} from '../../utils/format-utils';
+import {MAPBOX_STYLES, DATA_URI} from '../../constants/defaults';
+import {App} from 'website-examples/data-filter/app';
+
+export default class HighwayDemo extends Component {
```
Comment (ajduberstein) [inline website/src/components/demos/data-filter-demo.js]: Should the class name be changed?

**Pull request merged**